test-data/unit/check-classes.test

```diff
@@ -4131,6 +4131,20 @@ class C(str, str):
 [out]
```

Comment:
nit: maybe we remove testMroSetAfterError? Your new test case seems like a superset